gateway/src/main/scala/com/improving/app/gateway/domain/common/util.scala

```diff
@@ -0,0 +1,187 @@
+    lastName = contact.lastName,
+    emailAddress = contact.emailAddress,
+    phone = contact.phone,
+    userName = contact.userName
+  )
+
+  private def memberStateToGatewayMemberState(memberStatus: MemberStatus): GatewayMemberStatus =
+    if (memberStatus.isMemberStatusInitial) INITIAL_MEMBER_STATUS
+    else if (memberStatus.isMemberStatusActive) ACTIVE_MEMBER_STATUS
+    else if (memberStatus.isMemberStatusInactive) INACTIVE_MEMBER_STATUS
+    else if (memberStatus.isMemberStatusSuspended) SUSPENDED_MEMBER_STATUS
+    else TERMINATED_MEMBER_STATUS
+
+  private def memberMetaToGatewayMemberMeta(info: MemberMetaInfo): GatewayMemberMetaInfo = GatewayMemberMetaInfo(
+    createdOn = Instant.ofEpochSecond(info.createdOn.map(_.seconds).getOrElse(0L)),
+    createdBy = UUID.fromString(info.createdBy.getOrElse(MemberId.defaultInstance).id),
+    lastUpdated = Instant.ofEpochSecond(info.lastModifiedOn.map(_.seconds).getOrElse(0L)),
+    lastUpdatedBy = UUID.fromString(info.lastModifiedBy.getOrElse(MemberId.defaultInstance).id),
+    status = memberStateToGatewayMemberState(info.memberState)
+  )
+
+  private def notificationPreferenceToGatewayNotificationPreference(
+      notificationPreference: NotificationPreference
+  ): GatewayNotificationPreference = if (notificationPreference.isNotificationPreferenceEmail)
+    EMAIL_NOTIFICATION_PREFERENCE
+  else if (notificationPreference.isNotificationPreferenceSms) SMS_NOTIFICATION_PREFERENCE
+  else APPLICATION_NOTIFICATION_PREFERENCE
+
+  private def gatewayNotificationPreferenceToNotificationPreference(
+      notificationPreference: GatewayNotificationPreference
+  ): NotificationPreference = notificationPreference match {
+    case EMAIL_NOTIFICATION_PREFERENCE =>
+      NotificationPreference.fromValue(NotificationPreference.NOTIFICATION_PREFERENCE_EMAIL.value)
+    case SMS_NOTIFICATION_PREFERENCE =>
+      NotificationPreference.fromValue(NotificationPreference.NOTIFICATION_PREFERENCE_SMS.value)
+    case APPLICATION_NOTIFICATION_PREFERENCE =>
+      NotificationPreference.fromValue(NotificationPreference.NOTIFICATION_PREFERENCE_APPLICATION.value)
+  }
+
+  def memberResponseToGatewayEventResponse(
+      response: com.improving.app.member.domain.MemberResponse
+  ): MemberEventResponse =
+    if (response.asMessage.sealedValue.isMemberEventValue)
+      response.asMessage.getMemberEventValue.memberEvent match {
+        case response @ com.improving.app.member.domain.MemberRegistered(_, _, _, _, _) =>
+          MemberRegistered(
+            UUID.fromString(response.memberId.getOrElse(MemberId.defaultInstance).id),
+            memberInfoToGatewayMemberInfo(response.memberInfo.getOrElse(MemberInfo.defaultInstance)),
+            UUID.fromString(response.actingMember.getOrElse(MemberId.defaultInstance).id),
+            response.eventTime.getOrElse(Timestamp.defaultInstance).asJavaInstant
+          )
+        //  case response @ com.improving.app.member.domain.MemberActivated(_, _, _, _) =>
```

Comment:
Same deal here, why do we have commented out code? What work is needed before it can be uncommented?

Reply:
I copied code from Member service to get this started. I can possibly get rid of commented out code, but they will help with IA-149 (filling out MemberGateway), which I was originally going to finish out for this PR but then realized a minimal steel thread is what IA-150 was calling for

Reply:
Good idea